src/components/craft/translation.ts

```diff
@@ -158,6 +152,15 @@ const craftMessages = {
     solid: { id: 'craft.BackgroundStyleInput.solid', defaultMessage: '純色' },
     image: { id: 'craft.BackgroundStyleInput.image', defaultMessage: '圖片' },
   }),
+  SpaceStyleInput: defineMessages({
+    spacing: { id: 'craft.inputs.spacing', defaultMessage: 'spacing' },
+    margin: { id: 'craft.inputs.margin', defaultMessage: 'margin' },
+    padding: { id: 'craft.inputs.padding', defaultMessage: 'padding' },
+    top: { id: 'craft.label.top', defaultMessage: 'top' },
+    right: { id: 'craft.label.right', defaultMessage: 'right' },
+    bottom: { id: 'craft.label.bottom', defaultMessage: 'bottom' },
+    left: { id: 'craft.label.left', defaultMessage: 'left' },
+  }),
```

Comment:
id should be craft.SpaceStyleInput.xxx

Reply:
@sunnylochun I had fixed. You can see commit below this comment